docs/Doxyfile.in

```diff
@@ -467,31 +467,31 @@ LOOKUP_CACHE_SIZE      = 0
 # normally produced when WARNINGS is set to YES.
 # The default value is: NO.
 
-EXTRACT_ALL            = NO
+EXTRACT_ALL            = YES
```

Comment:
Haaa it turns out that these things were just not being generated... Thanks doxygen for removing stuff.